app/models/category.rb

```diff
@@ -0,0 +1,10 @@
+class Category < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :category_payments
+  has_many :payments, through: :category_payments
+  has_one_attached :icon
+
+  def recent_payments
+    payments.order('id ASC')
+  end
```

Comment:
- [ ] Your payments should be sorted in order of the most recent. So from the image below `Football` should come before `Shin Guard` since (from the time details) I created it last. Kindly fix that.

  ![jeff-order](https://user-images.githubusercontent.com/36542986/171870955-9f28207b-1801-4a64-b579-f717afb2b680.jpg)

Reply:
Fixed